--- src/driver/reset_device.c.orig      2007-03-20 05:33:29.000000000 -0500
+++ src/driver/reset_device.c   2009-05-07 10:41:28.331560796 -0500
@@ -32,14 +32,14 @@
     {
       DEBUG_ERROR("reset_device(): IOCTL_INTERNAL_USB_RESET_PORT failed: "
                   "status: 0x%x", status);
-    }

-  status = call_usbd(dev, NULL, IOCTL_INTERNAL_USB_CYCLE_PORT, timeout);
+      status = call_usbd(dev, NULL, IOCTL_INTERNAL_USB_CYCLE_PORT, timeout);

-  if(!NT_SUCCESS(status))
-    {
-      DEBUG_ERROR("reset_device(): IOCTL_INTERNAL_USB_CYCLE_PORT failed: "
-                  "status: 0x%x", status);
+      if(!NT_SUCCESS(status))
+        {
+          DEBUG_ERROR("reset_device(): IOCTL_INTERNAL_USB_CYCLE_PORT failed: "
+                      "status: 0x%x", status);
+        }
     }

   return status;
